generator/config/aggregation-stages/densify.yaml

```diff
@@ -0,0 +1,32 @@
+# $schema: ../schema.json
+name: $densify
+category:
+    - Stage
+link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/densify/'
+type:
+    - Stage
+encode: object
+description: |
+    Creates new documents in a sequence of documents where certain values in a field are missing.
+arguments:
+    -
+        name: field
+        type:
+            - FieldPath
+        description: |
+            The field to densify. The values of the specified field must either be all numeric values or all dates.
+            Documents that do not contain the specified field continue through the pipeline unmodified.
+            To specify a <field> in an embedded document or in an array, use dot notation.
+    -
+        name: partitionByFields
+        type:
+            - array<string>
+        optional: true
+        description: |
+            The field(s) that will be used as the partition keys.
+    -
+        name: range
+        type:
+            - Range
```

Comment:
`range` model is a plain object for now. Not modelized.

Reply:
For cases where we are choosing not to model the object, can we incorporate links in the YAML so that we get `@see` references in the generated code? Then, if folks happen to view the generated operator file they can be directed to the server documentation.

Reply:
I added a `@see` comment with the link to the doc on each operator class header.